bash_completion

```diff
@@ -1694,7 +1694,7 @@ _comp_compgen_help()
     for _line in "${_lines[@]}"; do
         [[ $_line == *([[:blank:]])-* ]] || continue
         # transform "-f FOO, --foo=FOO" to "-f , --foo=FOO" etc
-        while [[ $_line =~ ((^|[^-])-[A-Za-z0-9?][[:space:]]+)\[?[A-Z0-9]+([,_-]+[A-Z0-9]+)?(\.\.+)?\]? ]]; do
+        while [[ $_line =~ ((^|[^-])-[A-Za-z0-9?][[:space:]]+)(\[,?)?(\+\]?)?[A-Z0-9]+([,_-]+[A-Z0-9]+)?(\.\.+)?\]* ]]; do
```

Comment:
Does this correctly handle `MODE[,STATUS[,STATUS2]]`? `\[` only appears at the beginning and is not contained in any repetition (like `*`, `+`, and `{n,m}`), so `\[` is allowed only at the beginning.

Also, the special rule for `[+]`  seems specific to the case of `smartctl`. Or is this pattern `[+]` common for the output of `--help`? What does it actually mean? Is it a literal `+`?

Reply:
Thanks.